espanol/transcender/destino.md

```diff
@@ -0,0 +1,5 @@
+Justo antes de ver a Beatito había estado con Magola. El medio de una guerra no era el mejor momento para probar hongos alucinogénos por primera vez.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.